src/vidala.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
   rules: {
-    'no-restricted-imports': ['error', { 'paths': ['ViDaLa'] }]
+    'no-restricted-imports': ['error', { "patterns": ["ViDaLa/*"] }]
```

Comment:
Bitte single quotes verwenden.
```suggestion
    'no-restricted-imports': ['error', { 'patterns': ['ViDaLa/*'] }]
```

Reply:
was lukas sagt!